lib/tutorials/en_US/serving-files.md

```diff
@@ -171,4 +191,51 @@ server.route({
     }
 });
 ```
-Now a request to `/` will reply with HTML showing the contents of the directory.  When using the directory handler with listing enabled, by default hidden files will not be shown in the listing. That can be changed by setting the `showHidden` option to `true`. Like the file handler, the directory handler also has a `lookupCompressed` option to serve precompressed files when possible. You can also set a `defaultExtension` that will be appended to requests if the original path is not found. This means that a request for `/bacon` will also try the file `/bacon.html`.
+Now a request to `/` will reply with HTML showing the contents of the directory. When using the directory handler with listing enabled, by default hidden files will not be shown in the listing. That can be changed by setting the `showHidden` option to `true`. Like the file handler, the directory handler also has a `lookupCompressed` option to serve precompressed files when possible. You can also set a `defaultExtension` that will be appended to requests if the original path is not found. This means that a request for `/bacon` will also try the file `/bacon.html`.
+
+## <a name="fileserver" /> Static file server
+
+One common case for serving static content is setting up a file server. The following example shows how to setup a basic file serve in hapi:
+
+```js
+const Path = require('path');
+const Hapi = require('hapi');
+const Inert = require('inert');
+
+const init = async () => {
+
+    const server = new Hapi.Server({
+        port: 3000,
+        routes: {
+            files: {
+                relativeTo: Path.join(__dirname, 'public')
+            }
+        }
+    });
+
+    await server.register(Inert);
+
+    server.route({
+        method: 'GET',
+        path: '/{param*}',
+        handler: {
+            directory: {
+                path: '.',
+                redirectToSlash: true
+            }
+        }
+    });
+
+    await server.start();
+
+    console.log('Server running at:', server.info.uri);
+};
+
+init();
+```
+The first thing we do is require both `inert` and `path`. As you will see, we will need both of these to get our file server up and running.
+
+Next, we configure `server.options.routes`. We set the location the server will look for the static files by setting the `relativeTo` option.  
+
+After our server is configured, we then register the `inert` plugin. This will allow us to have access to the `directory` handler, which will enable us to server our files. In the `directory` handler, we configure `path`, which is required, to look in the entire `public` directory which we specified in the `relativeTo` option. The second option is the `redirectToSlash` option. By setting this to `true`, we tell the server to redirect requests without trailing slashes to the same path with the missing slash.
```

Comment:
Isn't the last sentence supposed to be:
> By setting this to `true`, we tell the server to redirect requests without trailing slashes to the same path with **those with** the ~~missing~~ trailing slash.

Reply:
Def should be....don't know what i was writing, doesn't even make sense, lol.  Fixing now